contrib/test/ci/critest.yml

```diff
@@ -36,8 +36,15 @@
 - name: Print critest version
   command: critest --version
 
+- name: Generate image mirror config
+  copy:
+    dest: "{{ ansible_env.GOPATH }}/src/github.com/cri-o/cri-o/critest-images.yml"
+    content: |
+      defaultTestContainerImage: {{ critest_mirror_repo }}/busybox:1
+      webServerTestImage: {{ critest_mirror_repo }}/nginx:1.18
+
 - name: run critest validation
-  shell: "critest --report-dir={{ artifacts }} --runtime-endpoint unix:///var/run/crio/crio.sock --image-endpoint unix:///var/run/crio/crio.sock --ginkgo.flakeAttempts=3"
+  shell: "critest --report-dir={{ artifacts }} --runtime-endpoint unix:///var/run/crio/crio.sock --image-endpoint unix:///var/run/crio/crio.sock --ginkgo.flakeAttempts=3 --test-images-file=critest-images.yml"
```

Comment:
I wasn't able to get critest to work with "critest --test-images-file=file.yaml", only with "critest test-images-file file.yaml", is this right?

Reply:
yeah, this is right... `critest --test-images-file=blah` fails locally when it can't find the file to load